app/assets/javascripts/discourse/app/components/badge-card.hbs

```diff
@@ -46,6 +39,17 @@
     <div class="badge-info-item">
       <h3><a href={{this.url}} class="badge-link">{{this.badge.name}}</a></h3>
       <div class="badge-summary">{{html-safe this.summary}}</div>
+
+      {{#if this.displayCount}}
+        <div class="badge-granted">
+          <a
+            href={{this.url}}
+            title={{i18n "badges.granted" count=this.displayCount}}
+            class="grant-count"
+          >{{this.displayCount}}x</a>
+          <span class="grant-text">{{i18n "badges.granted_text_only"}}</span>
```

Comment:
The "x" should be part of a translation. How about this?

```suggestion
          {{html-safe
            (i18n
              "badges.granted_with_count"
              count=this.displayCount
              badgeUrl=this.url
            )
          }}
```

See below for the suggested translation.

Reply:
Should it? It's not x as in alphabet here but x as in mathematical "times". So thought that would be okay?

Reply:
Nope, generally all punctuation marks and symbols should be part of a translation. I have no idea if the mathematical "times" symbol is used in every imaginable language, maybe it isn't so bad in this case? Anyway, by including the symbol in a translation we give translators the chance to replace the symbol or change the word order.